lib/format.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs')
+const path = require('path')
+
+const eslint = require('eslint')
+const prettier = require('prettier')
+const Linter = require('standard-engine').linter
+
+const logger = require('./logger')
+const util = require('./util')
+
+module.exports = function format (files) {
+  const linter = new Linter({
+    eslint,
+    eslintConfig: {
+      configFile: path.join(__dirname, './eslintrc.json')
+    }
+  })
+
+  if (files.length === 0) {
+    files = ['__mocks__', '__tests__', 'bin', 'lib', 'src', './']
+  }
+
+  const allFiles = util.cliFileEntriesToArray(files, false)
+  allFiles.forEach(file => {
```

Comment:
For our bigger projects this could take a significant amount of time. I'd be curious to see if there was an advantage to using the non-sync versions of `readFile`, `lintText` and `writeFile`. Thoughts?